cfgov/v1/atomic_elements/molecules.py

```diff
@@ -51,6 +51,10 @@ class ImageText2575(blocks.StructBlock):
     image = atoms.ImageBasic()
     links = blocks.ListBlock(atoms.Hyperlink(), required=False)
     has_rule = blocks.BooleanBlock(required=False)
+    should_link_image = blocks.BooleanBlock(default=False,
+                                            required=False,
+                                            help_text='Boolean indicating whether the image should be wrapped in a link tag.'
```

Comment:
Sorry to keep nitpicking this, but I think it should probably mention where it's getting the link from. How about something like, "Check this to link the image to the URL of the first link in the list, if there is a link." Or, if we move the option to the group, "Check this to link all images to the URL of the first link in their unit's list, if there is a link."